backend/src/main/scala/cromwell/backend/validation/RuntimeAttributesValidation.scala

```diff
@@ -57,4 +58,49 @@ object RuntimeAttributesValidation {
     }
   }
 
+  def validateMemory(value: Option[WdlValue], onMissingKey: () => ErrorOr[MemorySize]): ErrorOr[MemorySize] = {
```

Comment:
maybe just a thunk `onMissingKey: => ErrorOr[MemorySize]` to save parens and arrows and `apply`s? 😄